sklearn/datasets/base.py

```diff
@@ -255,14 +255,22 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        full description of the dataset. This is test text.
```

Comment:
I think I know the problem ;) sphinx uses your system installation of scikit-learn. Make sure that if you run python inside the doc folder and import scikit-learn, your dev version is imported.